src/components/shared/map/Map.tsx

```diff
@@ -339,11 +382,6 @@ const Map = ({
           zoomControl={false}
           minZoom={2}
           maxZoom={18}
-          whenReady={
```

Comment:
Why is this removed?  
This is the change that was made to initialize the seller map markers upon app load per [https://sharing.clickup.com/9015444408/t/h/86c0qgawh/PL7XY1MK0M64LW4](https://sharing.clickup.com/9015444408/t/h/86c0qgawh/PL7XY1MK0M64LW4).